rbs/MethodTypeToParserNode.cc

```diff
@@ -218,16 +292,19 @@ unique_ptr<parser::Node> MethodTypeToParserNode::methodSignature(const parser::N
 
         auto loc = declaration.typeLocFromRange(restKeywords->location->rg);
         auto node = (rbs_types_function_param_t *)restKeywords;
-        auto arg = RBSArg{loc, node->name, node->type};
+        auto arg = RBSArg{loc, node->name, node->type, RBSArg::Kind::RestKeyword};
         args.emplace_back(arg);
     }
 
     // Collect block
 
     auto *block = node.block;
     if (block) {
-        // TODO: RBS doesn't have location on blocks yet
-        auto arg = RBSArg{fullTypeLoc, nullptr, (rbs_node_t *)block};
+        auto loc = declaration.typeLocFromRange(block->base.location->rg);
+        // TODO: fix block location in RBS parser
+        loc.beginLoc = loc.beginLoc + 1;
+        loc.endLoc = loc.endLoc + 2;
```

Comment:
The location given by the RBS parser is incorrectly offset, I'll fix this upstream and remove this adjustment in a follow up.

Reply:
https://github.com/ruby/rbs/pull/2456